source/code/plugins/filter_linux_update_run_progress.rb

```diff
@@ -34,7 +34,11 @@ def shutdown
 
     def filter(tag, time, record)
       linuxUpdates = LinuxUpdates.new(@log, @current_update_run_file)
-      return linuxUpdates.process_update_run(record, tag, @hostname, time)
+      if tag == "oms.yum_update_progress"
```

Comment:
if you following the tag convention, you just need to check the last section of the tag to determine the processor.